test_vm/src/lib.rs

```diff
@@ -678,7 +678,7 @@ impl<'invocation, 'bs> Runtime<&'bs MemoryBlockstore> for InvocationCtx<'invocat
                 ));
             }
         }
-        let addr = Address::new_id(actor_id);
+        let addr = Address::new_id(*actor_id);
```

Comment:
As you mention in PR comments we need to change the sdk to avoid this.  Every reachable code path should get here so until we change the fvm interface I don't think this will improve performance.